app/grandchallenge/challenges/models.py

```diff
@@ -452,7 +452,7 @@ def save(self, *args, **kwargs):
             self.create_default_phases()
             send_challenge_created_email(self)
 
-        if self.hidden != self._hidden_orig:
+        if adding or self.hidden != self._hidden_orig:
```

Comment:
For my understanding, can you explain why this is added? Nothing else is changed with the Challenge model, right?

Reply:
This would fix a bug if you create a challenge with arguments of `hidden=False`, we don't actually do that anywhere but maybe one day we would. I hit that problem in the tests with public Jobs so figured to fix it here too.